### ISSUE TITLE
[accessibility] update accessible names/roles for images

### ISSUE THREAD

**Issue by ckundo** [jamstack/jamstack.org#365]
👋 hi, I noticed that many of the images are missing accessible names via alt text. This is a problem for screen reader users, and in some cases dictation users. Rather than enumerate all the locations here, I'll open a PR here.

**Comment by ckundo** [comment]
I think the PR #366 should resolve the issue, for the future check out the [AccessLint GitHub App](https://github.com/apps/accesslint/installations/new), it'll execute code review to guard against future issues. it's free for open source projects.

<img width="968" alt="" src="https://user-images.githubusercontent.com/108163/95030120-52831b00-067b-11eb-9789-17cac93f3604.png">

**Comment by zachleat** [comment]
Closing as stale, pending some changes in #366.
